.github/workflows/test.yml

```diff
@@ -28,10 +28,10 @@ jobs:
       run: isort . --check --diff
     - name: Run unit tests
       run: python -m unittest
-    - uses: google-github-actions/auth@v0
-      with:
-        credentials_json: ${{ secrets.GCP_SA_KEY }}
-    - name: Set up Cloud SDK
-      uses: google-github-actions/setup-gcloud@v0
-    - name: Integration test - client
-      run: python -m unittest test/street2sat_utils/test_client.py
+    # - uses: google-github-actions/auth@v0
+    #   with:
+    #     credentials_json: ${{ secrets.GCP_SA_KEY }}
+    # - name: Set up Cloud SDK
+    #   uses: google-github-actions/setup-gcloud@v0
+    # - name: Integration test - client
+    #   run: python -m unittest test/street2sat_utils/test_client.py
```

Comment:
You could just remove this line for now and add it as an additional Github issue